### ISSUE TITLE
unable to review history if speak command keys is enabled

### ISSUE THREAD

**Issue by cartertemm** [jscholes/nvda-speech-history#1]
when enabling speak command keys (NVDA+F4) and attempting to  review speech history we are unable to get anything. This is due to the pressed key being spoken, therefore interrupting and setting us back to the most recent item.

**Comment by jscholes** [comment]
@cartertemm I tested this and it seems to work fine with v2022.1, so it probably got fixed at some point.  Closing; let me know if you still have the problem.
